ci/terraform/oidc/dynamo-policies.tf

```diff
@@ -0,0 +1,135 @@
+data "aws_dynamodb_table" "user_credentials_table" {
+  name = "${var.environment}-user-credentials"
+}
+
+data "aws_dynamodb_table" "user_profile_table" {
+  name = "${var.environment}-user-profile"
+}
+
+data "aws_dynamodb_table" "client_registry_table" {
+  name = "${var.environment}-client-registry"
+}
+
+data "aws_dynamodb_table" "spot_credential_table" {
+  name = "${var.environment}-spot-credential"
+}
+
+data "aws_iam_policy_document" "dynamo_access_policy_document" {
+  count = var.use_localstack ? 0 : 1
```

Comment:
Personally, I would ditch all of these now, we don't need them as we don't regularly deploy to localstack and its OK to deploy to localstack anyway (they just get ignored).

Reply:
Good idea. I have updated @chrisclayson